docs/errors/err_ngrok_108.mdx

```diff
@@ -14,6 +14,7 @@ Your account is limited to `MAX` simultaneous ngrok agent session`PLURAL`.
 
 ### Further help
 If you're having trouble resolving this error, please reach out to [support@ngrok.com](mailto:support@ngrok.com?subject=Help%20with%20ERR_NGROK_108)
+If you're on a paid plan and would like to purchase another agent, you can go to your [billing settings](https://dashboard.ngrok.com/billing) in the ngrok dashboard and enable overages in the toggle. Then, you will be able to add another agent to your account. 
```

Comment:
We should add an additional newline above this line.

Also, should we add a line about upgrading to a paid plan?